aiohttp/web_response.py

```diff
@@ -88,6 +88,7 @@ def __init__(
         self._keep_alive = None  # type: Optional[bool]
         self._chunked = False
         self._compression = False
+        self._strategy = None  # type: int
```

Comment:
The comments for types is really old, let's use annotations for new code (`self._strategy: int = ...`).

But, you can't assign `None` to a type of `int`, use `Optional[]`, like the other types.